elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -0,0 +1,414 @@
+import prairielearn as pl
+import lxml.html
+import random
+import chevron
+import base64
+import os
+import json
+
+PL_ANSWER_DEFAULT_CORRECTNESS = True
+PL_ANSWER_DEFAULT_INDENT = '-1'
+CHECK_PL_ANSWER_INDENTION_DEFAULT = False
+SHUFFLE_MCQ_OPTIONS_DEFAULT = False
+DEFAULT_GRADING_MODE = 'ordered'
+DEFAULT_SOLUTION_PLACEMENT = 'right'
```

Comment:
Please use the same convention for _DEFAULT variable names as other files: https://github.com/PrairieLearn/PrairieLearn/blob/4ff1fd46b278b958ea554d43ebbe73959aca26e6/elements/pl-code/pl-code.py#L19

That is, they should be called `<VAR_NAME>_DEFAULT`, and `VAR_NAME` should be a [SCREAMING_SNAKE_CASE](https://en.wikipedia.org/wiki/Snake_case) version of the lower-kebab-case attribute name. For example, rather than `DEFAULT_GRADING_MODE` we should use `GRADING_METHOD_DEFAULT` to put `DEFAULT` at the end and match `grading-method` (not `-mode`).

Reply:
fixed